### PR TITLE
architecture_nb first term

### PR DESCRIPTION
**Pull request by henrikalbrecht**
Added missing arch_nb information
added c4 model pngs
added anchor to sys wide requirements to reference them properly

### CODE REVIEW

File: docs/architecture/images/Container.png
Comment (don-ny) [inline docs/architecture/images/Container.png:1]: please update system name to "VM-Vergabeverwaltungssystem"

File: docs/architecture/images/Context.png
Comment (don-ny) [inline docs/architecture/images/Context.png:1]: please update system name to "VM-Vergabeverwaltungssystem"

File: docs/architecture/architecture_notebook.adoc [hunk 1]
```diff
@@ -53,17 +53,21 @@ https://www2.htw-dresden.de/~anke/openup/core.tech.common.extend_supp/guidances/
 
 //Beschreiben Sie den Zweck, Eigenschaften und Funktion der Architekturmechanismen.
 
-. Architekturmechanismus 1
-. Architekturmechanismus 2
+. Um den Zugriff Unbefugter zu verhindern xref:../requirements/system-wide_requirements.adoc#NFAF1[(NFAF-1)] nutzen wir eine Kombination aus Nutzername und Passwort zur Authentifizierung des Vergabe-Verwalters.
+
+. Während der Analyse haben wir die Entscheidung getroffen die Daten persistent vorzuhalten.xref:../requirements/system-wide_requirements.adoc#NFAF2[(NFAF-2)] Die aktuelle Implementierung beruht auf einer SQLite Datenbank, da diese von Django direkt mitgeliefert wird. In Zukunft wird diese gegen eine PostgreSQL Datenbank ausgetauscht um den Anforderungen des Projektgebers zu entsprechen.
```
Comment (don-ny) [inline docs/architecture/architecture_notebook.adoc:58]:
```suggestion
. Während der Analyse haben wir die Entscheidung getroffen, die Daten persistent vorzuhalten. xref:../requirements/system-wide_requirements.adoc#NFAF2[(NFAF-2)] Die aktuelle Implementierung beruht auf einer SQLite Datenbank, da diese von Django direkt mitgeliefert wird. In Zukunft wird diese gegen eine PostgreSQL Datenbank ausgetauscht, um den Anforderungen des Projektgebers zu entsprechen. 
```

File: docs/requirements/system-wide_requirements.adoc [hunk 2]
```diff
@@ -19,8 +19,13 @@ In diesem Dokument werden die systemweiten Anforderungen für das Projekt {proje
 == Systemweite funktionale Anforderungen (Functionality)
 //Angabe von system-weiten funktionalen Anforderungen, die nicht als Use Cases ausgedrückt werden können. Beispiele sind Drucken, Berichte, Authentifizierung, Änderungsverfolgung (Auditing), zeitgesteuerte Aktivitäten (Scheduling), Sicherheit / Maßnahmen zum Datenschutz
 * *NFAF-1*: Authentifizierung
+[[NFAF1]]
 **  Um den Zugriff durch Unbefugte auf das System zu verhindern, muss eine Anmeldung an das System mittels Nutzername und Passwort erfolgen.
 
+* *NFAF-2*: Persistenz
+[[NFAF2]]
+** Das System muss in der Lage sein gespeicherte Daten nach einem Systemausfall oder Neustart wiederherzustellen. Zur Überprüfung wird die Datenbank mit Testdaten gefüllt. Anschließend wird das System ausgeschaltet und überprüft ob die Testdaten noch vorhanden sind. Wenn dies so ist, dann ist die Anforderung erfüllt.
```
Comment (don-ny) [inline docs/requirements/system-wide_requirements.adoc:27]:
```suggestion
** Das System muss in der Lage sein, gespeicherte Daten nach einem Systemausfall oder Neustart wiederherzustellen. Zur Überprüfung wird die Datenbank mit Testdaten gefüllt. Anschließend wird das System ausgeschaltet und überprüft, ob die Testdaten noch vorhanden sind. Wenn dies so ist, dann ist die Anforderung erfüllt.
```

**Review by don-ny — CHANGES_REQUESTED** [review]
mainly comma corrections <3

**Comment by henrikalbrecht** [comment]
Incorporated requested changes and updated the domain model according to MR Zs advise

**Review by don-ny — APPROVED** [review]

**Pull request merged**